engine/schema/src/main/resources/META-INF/db/schema-41720to41800.sql

```diff
@@ -869,3 +869,6 @@ WHERE   usage_unit = 'Policy-Month';
 
 -- delete configuration task.cleanup.retry.interval #6910
 DELETE FROM `cloud`.`configuration` WHERE name='task.cleanup.retry.interval';
+
+--- #6888 add index to speed up querying IPs in the network-tab
+CALL `cloud`.`IDEMPOTENT_CREATE_UNIQUE_INDEX`('uk_user_ip_address_state','user_ip_address', '(state)');
```

Comment:
if we add two shared  networks with same ip range but different vlans, will it be broken ?
they  have the same ip addresses and state (Free by default).

Reply:
:face_with_spiral_eyes: right, it shouldn´t be unique. I start over really thinking this through a bit.